City-of-Helsinki/berth-reservations-admin
# VEN-636 | Use Coat of Arms Blue in Select, fix line-height with required indicator

## Description :sparkles:

* Use Coat of Arms Blue in Select, fix line-height with required indicator

## Issues :bug:

### Closes :no_good_woman:

* [VEN-636](https://helsinkisolutionoffice.atlassian.net/browse/VEN-636): Use HDS drop down

## Testing :alembic:

### Automated tests :gear:️

* Updated snapshots with new className `select`

### Manual testing :construction_worker_man:

* Verify that the color is now correct
* Verify that the slight vertical difference in create/edit berth is now corrected
